samples/js/text_generation/README.md

```diff
@@ -92,6 +92,94 @@ Recommended models: Qwen/Qwen2.5-3B-Instruct, Qwen/Qwen2.5-7B-Instruct
   node react_sample.js model_dir
   ```
 
+### 6. Structured Output Sample (`structured_output_sample`)
```

Comment:
Please run this sample in https://github.com/openvinotoolkit/openvino.genai/blob/master/tests/python_tests/samples/test_structured_output_sample.py

Reply:
I tested it, and Python and JavaScript returned similar but not identical results. This occurs because they use different libraries and generate different prompts.
I will fix it and add this test in a separate PR.